results.pm

```diff
@@ -439,4 +443,39 @@ sub writelog
 	close($fh);
 }
 
+sub json_filter
+{
+	my ($filter, $logs) = @_;
+	my $filter_output;
+	$filter_output = openqa_filter($logs) if $filter == 'openqa';
```

Comment:
I think we need to add some (warning ? die ?) here in case not supported filter will be provided

Reply:
i added a warning